tests/js/roots/ecmascript/index.rst

```diff
@@ -0,0 +1,8 @@
+ECMAScript
+----------
+
+This is a sample JavaScript (aka ``ECMAScript``) project used to generate a search engine index fixture.
+
+.. js:attribute:: Object.__proto__
```

Comment:
Should we test this on a file *without* `__proto__` in it to make sure we don't match?

Reply:
That probably does make sense too, yep - thanks :+1:

Reply:
(in fact -- no probably about it -- adding a test like that definitely does make sense.  ensuring that no results are mistakenly reported when the term `__proto__` does not exist in the term index is the core of the bug)